config/internal_router/gen_keys.sh

```diff
@@ -52,11 +62,35 @@ PublicKey = $router_pubkey
 AllowedIPs = $GAME_NETWORK, $INTERNAL_NETWORK
 Endpoint = $ROUTER_ENDPOINT
 PersistentKeepalive = 15
+
+[Peer]
+PublicKey = $elk_pubkey
+AllowedIPs = $ELK_ADDRESS
+PersistentKeepalive = 15
+EOF
+)"
+
+elk_conf="$( cat <<-EOF
+[Interface]
+Address = $ELK_ADDRESS
+PrivateKey = $elk_privkey
+ListenPort = 51821
+
+[Peer]
+PublicKey = $router_pubkey
+AllowedIPs = $INTERNAL_NETWORK
+Endpoint = $ROUTER_ENDPOINT
+PersistentKeepalive = 15
+
+[Peer]
+PublicKey = $engine_pubkey
+AllowedIPs = $ENGINE_ADDRESS
+PersistentKeepalive = 15
 EOF
 )"
 
 mkdir -p clients
-for team_id in $(seq 1 "$1"); do
+for team_id in $(seq 2 "$1"); do
```

Comment:
Now that I think about that that breaks the checker deployment completely, since the config files start with 2 whereas terraform expects them to start with 1

Reply:
changed that